os/drivers/audio/Kconfig

```diff
@@ -59,6 +59,7 @@ config ALC5658_INITVOLUME
 config ALC5658_INFLIGHT
 	int "ALC5658 maximum in-flight audio buffers"
 	default 2
+	depends on AUDIO_DRIVER_SPECIFIC_BUFFERS
 
 config ALC5658_MSG_PRIO
```

Comment:
good point
but additionaly you had better move msg_prio to below of ALC5658_NUM_BUFFERS to clean
{configs depend on specific buffer}
{another configs}

Reply:
Agreed. Rearranged now.